.github/workflows/ci-cd.yml

```diff
@@ -103,7 +112,7 @@ jobs:
       #     path: Artifacts/NuGet/
 
   cd:
-    if: needs.ci.outputs.is_default_branch == 'true' && needs.ci.outputs.latest_version != ''
+    if: needs.ci.outputs.latest_version != ''
```

Comment:
Removed `is_default_branch` condition, because it is logically unnecessary, since the step that dictates if the other condition `needs.ci.outputs.latest_version` is true or false, only executes when the workflow is triggered by a push commit in the default branch.